.github/workflows/frontend-e2e.yml

```
@@ -0,0 +1,122 @@
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5440:5432
+
+    steps:
+      - uses: actions/checkout@v4
+
+      # 0. Setup Python (for migrations)
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)